app/assets/javascripts/application.js

```diff
@@ -55,3 +55,10 @@ $('.to_date')
         $('.from_date').datepicker('setEndDate', FromEndDate);
     });
 });
+
+
+!function ($) {
+  $(function(){
+    $('#myCarousel').carousel()
+  })
+}(window.jQuery)
```

Comment:
Wrap an immediate function invocation in parens to assist the reader in understanding that the expression is the result of a function, and not the function itself.<br>Expected an assignment or function call and instead saw an expression.<br>Missing semicolon.